elephant/surrogates.py

```diff
@@ -0,0 +1,490 @@
+# -*- coding: utf-8 -*-
+"""
+docstring goes here.
+
+:copyright: Copyright 2014 by the Elephant team, see AUTHORS.txt.
```

Comment:
A question here: how do we handle authorship and acknowledgments? How to know who contributed what?